CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -25,6 +25,7 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
     private var lastCollectionViewSize: CGSize = CGSize.zero
     private var lastScrollDirection: UICollectionView.ScrollDirection!
     private var lastItemSize: CGSize = CGSize.zero
+    private var isFirstCellCentered: Bool!
```

Comment:
I feel like this should instead be named `shouldFirstCellBeCentered` instead. What do you think?